wyklady/wyklad2.txt

```diff
@@ -0,0 +1,156 @@
+deck_id: 3892092678
+deck_name: Sieci komputerowe MIMUW
+file_id: sik-wyklad-2
+
+
+1
+Czym są programy sieciowe?
+To programy uruchamiane na systemach końcowych komunikujące się ze sobą za pośrednictwem sieci.
+
+2
+Podaj główne typy architektur aplikacji sieciowych.
+Architektura klient-serwer, architektura peer-to-peer (P2P), architektura hybrydowa (mieszanka obu).
+
+3
+Co jest najważniejszą cechą protokołu TCP?
+Zapewnia on niezawodny transfer danych - bez błędów i w odpowiedniej kolejności.
+
+4
+Co oznacza, że protokół TCP jest zorientowany na połączenie?
+Składa się on z etapów: nawiązywania połączenia (faza negocjacji), wysyłania komunikatów i zakończenia połączenia.
+
+5
+Co określa protokół warstwy aplikacji?
+Typy komunikatów, składnię różnych typów komunikatów, semantykę i znaczenie pól, zasady i czas wysyłania i odbierania komunikatów.
```

Comment:
Może lepiej zamienić na listę? Wygodniej się to czyta, według Jacka \<br> działa jako znak nowej linii.